src/unix.rs

```diff
@@ -330,8 +334,22 @@ pub fn set_thread_priority_and_policy(
         // SCHED_DEADLINE policy requires its own syscall
         #[cfg(target_os = "linux")]
         ThreadSchedulePolicy::Realtime(RealtimeThreadSchedulePolicy::Deadline) => {
+            use std::convert::TryInto as _;
```

Comment:
I have never seen `use X as _;`. Is there any benefit from using it that way instead of just `use std::convert::TryInto`?

Reply:
It just includes the trait without polluting the namespace. There's no real upside to it, but no downside either if you're not manually implementing the trait.

Reply:
Thanks.